ember-flight-icons/tests/acceptance/index-test.js

```diff
@@ -9,9 +9,11 @@ module('Acceptance | icon index', function (hooks) {
 
   test('visiting / renders a list of icons', async function (assert) {
     await visit('/');
-    // added this because the icons are not rendering as quickly
-    await waitFor('.flight-icon', { timeout: 1000 });
+
+    // added timeout because the icons are not rendering as quickly
```

Comment:
@MelSumner @amyrlam do we have any idea why the icons are not rendering quickly (you can see this when you load the docs)? Is it something specific to how we are rendering them in the docs or is it inherent to the sprite? This makes me concerned that folks using the addon could experience a similar delay which would not be good.

Reply:
I don't see it loading that slowly in tests. Even running an `ember test --server` with a debugger, it was hard to say. I also ran some of the ember try:once commands locally

Does Vercel have caching out of the box? I could honestly set up Fastly for us in 30-60min, it would just be a straight up config with gzip (I think)

Reply:
I'll make a separate issue to better capture this

Reply:
https://github.com/hashicorp/flight/issues/226